source/ed/connectors/gtfs_parser.cpp

```diff
@@ -400,6 +402,7 @@ nm::StopArea* StopsGtfsHandler::build_stop_area(Data& data, const csv_row& row)
     }
     gtfs_data.stop_area_map[sa->uri] = sa;
     data.stop_areas.push_back(sa);
+    add_gtfs_stop_code(data, sa, row[code_c]);
```

Comment:
I do not believe adding this `gtfs_stop_code` when it's missing is correct. You do not know if the input NTFS has been generated from GTFS or from another data format.

So far, the object code `gtfs_stop_code` has not been removed on `transit_model` side. We can definitely let it until we're sure it doesn't impact Navitia consumers. The only important thing would be to communicate on Navitia side that it will eventually be replaced by `stop_code` (so consumers can change their code).